### DISCUSSION TITLE
Some problem about the rank.py in ianvs

### DISCUSSION THREAD

**Discussion by Yoda-wu** [kubeedge/ianvs#345] [Q&A] [answered]
i am using the incremental learning paradigm, while produce the rank table, some straght problem occus:
1. Pandas have no attribute  np

![image](https://github.com/kubeedge/ianvs/assets/64141194/755e8c3f-2721-4559-a02a-2468a2aa28af)

2. DataFrame object have no attribute "append" 

![image](https://github.com/kubeedge/ianvs/assets/64141194/ccd3a42d-1a12-46d7-8520-2daf76d0d8bf)

Is that cause by the different version of pandas? but i can not find any information about pd.np

**Comment by hsj576** [comment, marked as answer]
The first error may caused by the wrong version of pandas, you could use "pip install pandas==1.1.5" to install the correct version of pandas.
